content/pages/feed/feed.md

```diff
@@ -0,0 +1,40 @@
+---
+permalink: /feed.xml
+skip_index: true
+---
+{% assign selected_posts = collections.posts | reverse | slice:0, 10 -%}
+<feed xmlns="http://www.w3.org/2005/Atom">
+<link href="{{ site.host }}/feed.xml" rel="self" type="application/atom+xml"/>
+<link href="{{ site.host }}/" rel="alternate" type="text/html"/>
+<updated>{{ selected_posts.0.date | date_to_xmlschema }}</updated>
```

Comment:
change: Preference for indexed access style `[0]` instead of dot access `.0` for arrays

Reply:
Done